### PR TITLE
Adds styling for admin code csv page and also code csv generation modal, modifies existing style for modals, and adds svg assets.

### LINKED ISSUE #116: Design code csvs page and then style it
Should start with the list of code_csvs + buttons

### PR DESCRIPTION
**Pull request by lbkchen**
Resolves #116

### CODE REVIEW

File: app/assets/stylesheets/pages/code_csv_list_page.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.code-csv-container {
+  width: 66%;
+  margin-top: 30px;
+}
+
+.code-csv-card {
+  justify-content: space-between;
+  height: 60px;
+  padding: 15px 20px;
+  margin: 0 auto;
```
Comment (charlesx2013) [inline app/assets/stylesheets/pages/code_csv_list_page.scss]: you can collapse this into `margin: 10px auto`

File: app/assets/javascripts/admins/components/GenerateCodeCsvCourseCard.js [hunk 2]
```diff
@@ -16,28 +16,11 @@ class GenerateCodeCsvCourseCard extends React.Component {
...
-      fontSize: '24px'
-    }
-  }
-
   render() {
     return (
-      <div>
-        <h2 style={this.getStyle()} onClick={this.handleClick}>{this.props.course.name}</h2>
+      <div
+        className={`flex generate-code-csv-course-card ${this.state.selected ? ' active' : ''}`} onClick={this.handleClick}>
```
Comment (charlesx2013) [inline app/assets/javascripts/admins/components/GenerateCodeCsvCourseCard.js]: try splitting this up into multiple lines.

**Comment by charlesx2013** [comment]
Don't forget to add a photo!

**Comment by lbkchen** [comment]
screen shot 2016-11-17 at 9 28 27 pm
screen shot 2016-11-17 at 9 27 50 pm

**Comment by charlesx2013** [comment]
👍

**Pull request merged**